### DISCUSSION TITLE
[Blog post] Origin story of ScrollMaps

### DISCUSSION THREAD

**Discussion by mauricelam** [mauricelam/ScrollMaps#46] [General]
Read the full story here: https://mauricelam.github.io/2022/02/23/scrollmaps-origin-story.html

[![Screen Shot 2022-02-26 at 11 14 49 AM](https://user-images.githubusercontent.com/1264702/155856201-fa9f65c5-516f-43a4-9fa7-5717d32c16c1.png)](https://mauricelam.github.io/2022/02/23/scrollmaps-origin-story.html)

**Comment by joanjgm** [comment]
Hi, great addon, is there a safari extension available? the link on the homepage is broken

**Reply by mauricelam** [reply]
Unfortunately no. I had the Safari version available for a while, before Apple changed it so that I'll need to pay a yearly developer fee to publish the extension on the app store. Since I have no way to recoup that cost, I decided to pull the plug on it.
